hadoop-ozone/fault-injection-test/mini-chaos-tests/src/test/java/org/apache/hadoop/ozone/TestMiniChaosOzoneCluster.java

```diff
@@ -123,6 +133,8 @@ public void run() {
       cluster.startChaos(failureInterval, failureInterval, TimeUnit.SECONDS);
       loadGenerator.startIO(numMinutes, TimeUnit.MINUTES);
     } catch (Exception e) {
+      LOG.error("terminating with exception", e);
+      ExitUtil.terminate(-2, e);
```

Comment:
Can you please help to understand the motivation behind this? I thought that if an exception is thrown the exit code will be non-zero anyway (If something is wrong with the error handling, GenericCli might already have the solution. As I remember it has some generic error handling logic).

Reply:
done, moved to GenericCli.